### ISSUE TITLE
HBase fails to run on Travis CI 

### ISSUE THREAD

**Issue by madisonb** [istresearch/ansible-symphony#1]
The Hbase role doesn't seem to build correctly on Travis CI, thus causing the tests to fail if they are included. However, it builds correctly on the Vagrant VM.

**Comment by jasonrhaas** [comment]
It has something to do with the different in network interfaces between the Travis VM and the Vagrant VM we have set up.  For example, on the Vagrant VM the /etc/hosts file looks like:

```
127.0.0.1        localhost
192.168.33.101   vagrant-as-01  vas01
192.168.33.102   vagrant-as-02  vas02
```

As part of the `Vagrantfile` I copy over the `hosts` file and overwrite the /etc/hosts files with that information.  The 192 IP address is on eth1 in the Vagrant VM.  For the Travis Ubuntu VM, there isn't an eth1, it is trying to use eth0, which is also tied to localhost.  I think this is part of the problem although I still haven't been able to crack this issue.

The second point #2 here https://wiki.apache.org/hadoop/ConnectionRefused seems to hint at the problem we are having.

> Check that there isn't an entry for your hostname mapped to 127.0.0.1 or 127.0.1.1 in /etc/hosts (Ubuntu is notorious for this)

**Comment by jasonrhaas** [comment]
So I figured out one issue -- you can't have the **hbase-namenode-node** and **hbase-secondary-namenode-node** on the same host.  I took the secondary namenode out of the the **testing** inventory and was able to get past the initial hadoop error that was happening during the Hbase play `- name: ensure hdfs directory for hbase root is created`.

However, the Hbase region server fails to start (spawn error).  The region server spawns on port 60020.  I've seen this issue on another Ubuntu machine we had in production, but re-imaging the machine fixed the problem.  So right now I'm only seeing it on the Travis Trusty VM.

**Comment by andrewkcarter** [comment]
This pull request resolves the spawn error: #5
